app/src/main/kotlin/com/wire/android/ui/home/FeatureFlagState.kt

```diff
@@ -23,5 +23,7 @@ package com.wire.android.ui.home
 data class FeatureFlagState(
     val showFileSharingDialog: Boolean = false,
     val isFileSharingEnabledState: Boolean = true,
-    val showFileSharingRestrictedDialog: Boolean = false
+    val showFileSharingRestrictedDialog: Boolean = false,
+    val shouldShowGuestRoomLinkDialog: Boolean = false,
+    val isGuestRoomLinkEnabled: Boolean = true
```

Comment:
Should this be enabled by default? Shouldn't it start as false? 🤔

Reply:
No guest link are enabled by default unless a team wants to disable it.